### PR TITLE
Add Newtonsoft CamelCase config support

### LINKED ISSUE #209: camelCase serialisation in output binding.
https://docs.microsoft.com/en-us/aspnet/core/signalr/configuration?view=aspnetcore-5.0&tabs=dotnet#jsonmessagepack-serialization-options

If self hosting then it defaults to using camelCase, but it appears that the azure function version requires us to manually serialise the message ourselves.

Can either the default behaviour be changed to match the normal client/server defaults or some form of configuration added to enable specifying the behaviour?

### PR DESCRIPTION
**Pull request by Y-Sindo**
Allow to use "Azure:SignalR:HubProtocol:NewtonsoftJson:CamelCase=true" to enable camel case.
Fixed #209

### CODE REVIEW

File: src/SignalRServiceExtension/Constants.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ internal static class Constants
         public const string AzureSignalRConnectionStringName = "AzureSignalRConnectionString";
         public const string AzureSignalREndpoints = "Azure:SignalR:Endpoints";
         public const string AzureSignalRHubProtocol = "Azure:SignalR:HubProtocol";
+        public const string AzureSignalRNewtonsoftCamelCase = "Azure:SignalR:HubProtocol:Newtonsoft:CamelCase";
```
Comment (JialinXin) [inline src/SignalRServiceExtension/Constants.cs:11]: Would this config too deep?

Comment (JialinXin) [reply src/SignalRServiceExtension/Constants.cs:11]: `Azure:SignalR:HubProtocol` accepts `SystemJson` and `NewtonsoftJson`, is there relationship with this config?

Comment (Y-Sindo) [reply src/SignalRServiceExtension/Constants.cs:11]: `AzureSignalRNewtonsoftCamelCase` would take effect only when `Azure:SignalR:HubProtocol=NewtonsoftJson`.

File: src/SignalRServiceExtension/Constants.cs [hunk 2]
```diff
@@ -8,6 +8,7 @@ internal static class Constants
         public const string AzureSignalRConnectionStringName = "AzureSignalRConnectionString";
         public const string AzureSignalREndpoints = "Azure:SignalR:Endpoints";
         public const string AzureSignalRHubProtocol = "Azure:SignalR:HubProtocol";
+        public static string AzureSignalRNewtonsoftCamelCase = $"{AzureSignalRNewtonsoftCamelCase}:{HubProtocol.NewtonsoftJson}:CamelCase";
```
Comment (JialinXin) [inline src/SignalRServiceExtension/Constants.cs:11]: Typo. Update tests for better coverage.
> $"{AzureSignalRHubProtocol}:{HubProtocol.NewtonsoftJson}:CamelCase";

**Review by JialinXin — APPROVED** [review]

**Review by JialinXin — APPROVED** [review]

**Pull request merged**